rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -664,3 +730,280 @@ def test_applying_all_upgrade(globaldb: GlobalDBHandler):
+
+        # if the updated asset values are different from the packaged asset values, prepare their updates  # noqa: E501
+        if packaged_assets[identifier].as_dict() != updated_assets[identifier].as_dict():
+            updates: dict[str, dict[str, str]] = defaultdict(dict)  # mapping from table name to mapping from column name to value, that needs to update  # noqa: E501
+            fields_to_update: dict = {'underlying_tokens': []}
+            for key, packaged_field in packaged_assets[identifier].as_dict().items():
+                updated_field = getattr(updated_assets[identifier], key)
+                if packaged_field is not None and updated_field != packaged_field:
+                    updates[field_to_table[key]][key] = packaged_field
+                    fields_to_update[key] = packaged_field
+
+                elif updated_field is not None and packaged_field is None:
+                    missing_in_packaged_db.append(f'Asset: {identifier} ({key}) - {packaged_field} (packaged) != {updated_field} (asset update)')  # noqa: E501
+                    fields_to_update[key] = updated_field
+
+                else:
+                    fields_to_update[key] = packaged_field
+
+            if packaged_underlying_assets[identifier] != updated_underlying_assets[identifier]:
+                if len(updated_underlying_assets[identifier] - packaged_underlying_assets[identifier]) > 0:  # noqa: E501
+                    missing_in_packaged_db.append(f'Asset: {identifier} ({key}) - {packaged_field} (packaged) != {updated_field} (asset update)')  # noqa: E501  # pylint: disable=undefined-loop-variable
+
+                if len(underlying_tokens_to_update := updated_underlying_assets[identifier] - packaged_underlying_assets[identifier]) > 0:  # noqa: E501
+                    fields_to_update['underlying_tokens'] = [
+                        updated_assets[identifier].address
+                        for identifier in underlying_tokens_to_update
+                    ]
+
+            if len(updates) > 0:
+                missing_in_updates.append(ASSET_UPDATE.format(
+                    address=fields_to_update['address'],
+                    chain_name=ChainID.deserialize(fields_to_update['chain']).name,
+                    coingecko=fields_to_update['coingecko'],
+                    cryptocompare=fields_to_update['cryptocompare'],
+                    field_updates=dict(updates),
+                    protocol=fields_to_update['protocol'],
+                    underlying_token_addresses=fields_to_update['underlying_tokens'],
+                ))
+
+    missing_in_updates = [
+        str(update).replace('None', '')
+        for update in missing_in_updates
+    ]
+
+    if missing_in_packaged_db != []:
+        warn('\n'.join(missing_in_packaged_db))
+
+    if missing_in_updates != []:
+        pytest.fail('\n'.join(missing_in_updates))
+
+
+def test_oracle_ids_in_asset_collections(globaldb: 'GlobalDBHandler'):
```

Comment:
put this test in same place. They are both not DB upgrade but DB consistency checks.